src/IntercomContacts.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+namespace Intercom;
+
+use Http\Client\Exception;
+use stdClass;
+
+class IntercomContacts extends IntercomResource
+{
+    /**
+     * Creates a Contact.
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#create-contact
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function create(array $options)
+    {
+        return $this->client->post("contacts", $options);
+    }
+
+    /**
+     * Updates a Contact.
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#update-contact
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function update(array $options)
+    {
+        return $this->client->put("contacts", $options);
+    }
+
+    /**
+     * Lists Contacts.
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#list-contacts
+     * @param  array $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function getContacts(array $options)
+    {
+        return $this->client->get('contacts', $options);
+    }
+
+    /**
+     * Gets a single Contact based on the Intercom ID.
+     *
+     * @see    https://developers.intercom.com/intercom-api-reference/reference#get-contact
+     * @param  string $id
+     * @param  array  $options
+     * @return stdClass
+     * @throws Exception
+     */
+    public function getContact($id, $options = [])
```

Comment:
I'm a strongly typing fan 😅 

```suggestion
    public function getContact($id, array $options = [])
```